### DISCUSSION TITLE
120Ω needed on Nasa?  - physical layer

### DISCUSSION THREAD

**Discussion by johnnytolengo** [omerfaruk-aran/esphome_samsung_hvac_bus#129] [General]
Hi, I was looking the info about the Nasa protocol and there not information about the physical layer.
Nasa seems to uses a kind of multimaster communication, where all devices can write to the bus independently.

- Is 120Ω resistor needed on our device (M5STACK ATOM RS-485)?
- Do all the Samsung devices have a 120Ω resistor by default?
-  Bias resistors needed (pull-up & pull-down A B)?

![image](https://github.com/lanwin/esphome_samsung_ac/assets/5218298/90dfa668-e5b8-428b-bcc7-ca0bf3aff6a3)

**Comment by lanwin** [comment]
@johnnytolengo sorry, I have no Idea. I am not a hardware guy. But I did not added any resistor to make it work and non of the other users does.
